packages/flutter/lib/src/rendering/selection.dart

```diff
@@ -90,6 +90,13 @@ abstract class SelectionHandler implements ValueListenable<SelectionGeometry> {
   /// Return `null` if nothing is selected.
   SelectedContent? getSelectedContent();
 
+  /// Gets the [SelectedContentRange] representing the selected range in this object.
+  ///
+  /// When nothing is selected, subclasses should return a [SelectedContentRange.empty],
+  /// with a [SelectedContentRange.contentLength] that represents the length
+  /// of the content under it.
```

Comment:
Nit: "under it" => "under it in the widget tree"

Or something more specific like that.